ui/README.md

```diff
@@ -0,0 +1,64 @@
+# UI
+
+This directory is for the React (non-lite/AMP) UI.
+
+## Development server
+
+The easiest way to develop the UI is to run two local servers:
+
+1. Run the Django (backend) server as usual, at port 8000.
+2. Run the UI with `tools/ui run`. It will talk to the backend at port 8000.
+
+Changes you make to the JS and CSS will take effect when you refresh the page.
+If you make other changes (e.g., to install a new module or modify the Webpack
+config) you'll likely need to restart the server.
+
+Currently, it is necessary to change `USE_REACT_UI` in `app/main.py` to `True`.
```

Comment:
Can you use config to configure this?

Reply:
I could, but I think there's a small advantage to doing it with a constant like this: when we want to make the switch for real in prod, we can upload a new version to AE with the boolean flipped and then use AppEngine versioning to put a small percentage of traffic on the new version at first, as a canary.

Reply:
Understood. It's up to you, but I found it's often annoying when I always need to flip a constant value locally and make sure to revert it before submitting it.

Maybe define useReactUI() function which is backed by a config for now, and switch it to a constant once it's ready to ramp up? We can discuss later though.